frontend/src/components/core/Block/Block.tsx

```diff
@@ -167,22 +143,29 @@ class Block extends PureComponent<Props> {
   ): ReactNode | null {
     const element = reportElement.get("element")
 
-    if (element.get("type") === "empty") {
```

Comment:
Should we remove this?

Reply:
I removed it, right?

Reply:
I actually ket it, so that we don't have to pass around an `undefined` component.